README.adoc

```diff
@@ -359,6 +359,42 @@ There are also two shorthands for adding the attributes `options=""` and `cols="
 With the cursor inside the table, kbd:[+,cols+] adds the latter and kbd:[+,opts+] the former.
 
 
+==== Aligning
+
+When the https://github.com/godlygeek/tabular[Tabular Plugin] is
+installed (which is highly recommended when working with tables),
+alignment of tables is possible via kbd:[+,|+]. This applies only to tables
+that use the Bar kbd:[+|+] as separator between cells.
+
+Be aware that this only works for basic tables and certain features like
+cell spanning do not work well with it together. It won't break the table,
+but the cell spanning will not be visualized in the asciidoc sourcecode.
+
+.Before
+....
+[%header]
```

Comment:
The %<option> stuff is a nice feature but it's only noise in this context. I don't see how this adds any value for the reader. If it does it should be added in one fell swoop everywhere.